### DISCUSSION TITLE
APPCRASH  MSVCR120.dll, version: 12.0.40660.0,  Exception code: 0x40000015

### DISCUSSION THREAD

**Discussion by vadlexa** [cefsharp/CefSharp#3488] [General]
### Bug Report
- used versions:  49, 57, 63
- architecture x86 	
- .Net 4.x   
- Win7
- WinForms
-  Error is in random time:
      Faulting application name: FormUI.exe, version: 1.0.0.10, time stamp: 0xd907ce1e
      Faulting module name: MSVCR120.dll, version: 12.0.40660.0, time stamp: 0x577e0f1e
      Exception code: 0x40000015 
      Fault offset: 0x000a7656
      Faulting process id: 0x1344
      Faulting application start time: 0x01d72ac1fb6803cc
      Faulting application path: C:\Terminal\Logic\FormUI.CefSharp\FormUI.exe
      Faulting module path: C:\Windows\system32\MSVCR120.dll

Last method witch called in my app is IFrame.EvaluateScriptAsync  and then app crashed
I used  v49, v57. v63 and all versions have the same problems.

**Comment by amaitland** [comment]
> I used v49, v57. v63 and all versions have the same problems.

All of these versions are out of support. The current supported version is 89, does the problem reproduce with this version?
